lib/src/common/disposable.dart

```diff
@@ -112,6 +112,14 @@ class _ObservableTimer implements Timer {
 
   @override
   bool get isActive => _timer.isActive;
+
+  @override
+  int get tick {
```

Comment:
So is `tick` even a field on `Timer` in Dart 1? (i.e. is this `@override` valid in Dart 1?)

Reply:
No, the override results in a warning in Dart 1, but it still works.